generators/profilegen/src/profile_generator.rb

```diff
@@ -69,12 +68,9 @@ def generate_modules
       @spec_includes << @module_require
```

Comment:
could there be more than module_required? 
could there be several RPs pointing to the same module_required?